### PR TITLE
Allow extra_data and payload to be sent, too

### PR DESCRIPTION
**Pull request by mbrevda**
Any thoughts @paulstatezny? Did you specifically exclude this data?

### CODE REVIEW

File: src/Monolog/Handler/RollbarHandler.php [hunk 1]
```diff
@@ -54,7 +54,15 @@ public function __construct(RollbarNotifier $rollbarNotifier, $level = Logger::E
     protected function write(array $record)
     {
         if (isset($record['context']['exception']) && $record['context']['exception'] instanceof Exception) {
-            $this->rollbarNotifier->report_exception($record['context']['exception']);
+            $context = $record['context'];
+            unset($context['exception']);
+
+            $payload = isset($context['payload']) ? $context['payload'] : [];
+            if ($context['payload']) {
```
Comment (paulstatezny) [inline src/Monolog/Handler/RollbarHandler.php]: Should this be `if (isset($context['payload']))`?

When the `isset` check above fails, `if ($context['payload'])` will cause an `undefined index` error, correct?

**Comment by paulstatezny** [comment]
@mbrevda Sorry for the delay. I did not intentionally exclude this data. Just wasn't this thorough. Thanks for building on it. :smile:

**Comment by mbrevda** [comment]
@Seldaek - can we get this merged?

**Comment by Seldaek** [comment]
Looks good so let's merge, I'm in holidays so not watching github too closely

**Pull request merged**